tedana/utils.py

```diff
@@ -150,6 +153,12 @@ def make_adaptive_mask(data, mask=None, minimum=True, getsum=False):
         # if the user has supplied a binary mask
         mask = load_image(mask).astype(bool)
         masksum = masksum * mask
+        # reduce mask based on masksum
+        if np.any(masksum[mask] == 0):
+            n_bad_voxels = np.sum(masksum[mask] == 0)
+            LGR.warning('{0} voxels in user-defined mask do not have good '
+                        'signal. Removing voxels from mask.'.format(n_bad_voxels))
```

Comment:
Could we add a "#TODO" about asking them to visually inspect the mask ? This is something we should include in the visual reports, eventually.

Reply:
I can do that, although we aren't actually writing out the masks at the moment. I was planning to incorporate writing out the masks and other intermediate outputs into `verbose` in a future PR, but can at least write out the masks with `verbose` in this one.

Reply:
Oh sorry I misinterpreted your comment. I'll add a comment to the code.